tests/src/skills/can_divide_tests.cpp

```diff
@@ -44,15 +47,17 @@ struct a_can_divide_with_int_and_double_type
     using stronk::stronk;
 };
 
-TEST(can_divide_with, can_divide_with_integers_and_other_types)
+TEST_SUITE("can_divide_with")
 {
-    auto val = a_can_divide_with_int_and_double_type {200};
-    val /= 10;
-    val = val / 10;
-    EXPECT_EQ(val, a_can_divide_with_int_and_double_type {2}) << val.unwrap<a_can_divide_with_int_and_double_type>();
-    val /= 0.1;
-    val = val / 0.1;
-    EXPECT_EQ(val, a_can_divide_with_int_and_double_type {200});
+    TEST_CASE("can_divide_with_integers_and_other_types")
+    {
+        auto val = a_can_divide_with_int_and_double_type {200};
+        val /= 10;
+        val = val / 10;
+        CHECK_EQ(val, a_can_divide_with_int_and_double_type {2});
```

Comment:
You can add an INFO call above, if you want to keep the print.
Some of the CHECK functions also have a `_MESSAGE` suffix, that works a bit like the previous code. I'm unsure if `CHECK_EQ_MESSAGE` exists though.